amp_conf/htdocs/admin/libraries/Console/Chown.class.php

```diff
@@ -149,6 +149,9 @@ protected function execute(InputInterface $input, OutputInterface $output, $quie
 			$this->modfiles['framework'][] = array('type' => 'execdir',
 			'path' => $AMPBIN,
 			'perms' => 0755);
+			$this->modfiles['framework'][] = array('type' => 'dir',
```

Comment:
Woah, good point. I totally missed that. That do this:

chown -R asterisk.asterisk /usr/sbin

Pretty sure we don't want to do that